src/libs/functions/urlHelpers.js

```diff
@@ -62,6 +62,21 @@ export const isSameCozy = ({
       destinationUrl,
       subdomainType
     )
+
+    if (subdomainType === 'nested') {
+      // http://dev.10-0-2-2.nip.io:8080 to http://contacts.dev.10-0-2-2.nip.io:8080/
+      const nestedName = currentUrlData.slug // dev
```

Comment:
For me, we should not have a slug there. It should be empty for `http://dev.10-0-2-2.nip.io:8080` but maybe you don't know that `dev` is not a slug ATM. @Ldoppea don't we have a method in cozy-client that handles that?

Reply:
I think I understand the problem here and seems like we are not fixing it on the more optimal place.

Here the method is based on `cozy-client/deconstructCozyWebLinkWithSlug`. The method's name explicitly states that it expects to have URLs with slugs (this is the meaning of `WithSlug`). If the provided URL doesn't contain any slug then we won't be able to correctly deconstruct the link.

Here the `isSameCozy` method is called only from `openApp()` that uses `client.getStackClient().uri` for cozyUrl, so we are sure that the url WON'T contain any slug.

Instead of adding this specific code I propose to either:
- Enforce a slug from the caller (i.e use `generateWebLink` in `openApp` or something similar)
or
- Implement a method in cozy-client called `deconstructCozyWebLinkWithoutSlug`